9.3/paper.tex

```diff
@@ -918,10 +918,99 @@ \subsection{Evaluation and integration at arbitrary points}
 
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\subsection{Terminal based toolchain installer}
+\subsection{Source based toolchain installer}
 \label{subsec:candi}
 
-Half a page up to a page on candi.
+The requirement to download, compile and install \dealii{} and its dependencies from
+source is of mayor interest for many \dealii{}-users.
+%
+This can be of importance on arbitrary Linux-based operating systems for which
+binary-install package aren't available or on compute clusters and other machines
+without root privileges for the user to install system dependencies.
+
+The source based installation of \dealii{} or complex toolchains of multiple libraries
```

Comment:
```suggestion
The source based installation of \dealii{} and it's many dependent libraries
```

Reply:
It should be "its", not "it's". I'll fix that myself.